src/mcts/node.h

```diff
@@ -336,8 +337,10 @@ class EdgeAndNode {
   Node* node() const { return node_; }
 
   // Proxy functions for easier access to node/edge.
-  float GetQ(float default_q) const {
-    return (node_ && node_->GetN() > 0) ? node_->GetQ() : default_q;
+  float GetQ(float default_q, bool logit_q = false) const {
```

Comment:
It may make sense to remove `= false` default parameter in order no to forget passing correct value in some important place.
But if it makes lots of places in the code uglier, it's fine to keep it.

Reply:
There are quite a few places where GetQ is called but only a few places where logit should be applied so I think this way is safer and cleaner.